default.nix

```diff
@@ -0,0 +1,23 @@
+{
+  lib,
+  rustPlatform,
+}: let
+  version = "0.1.0";
+  owner = "ynqa";
+  repo = "sig";
+in
+  rustPlatform.buildRustPackage {
+    pname = "sig";
+    inherit version;
+
+    src = ./.;
+
+		cargoHash = "sha256-yz/DPJJxtsHpJLpTMAYfsq9miIs48F+FeSnmkQ707uA=";
```

Comment:
@Dich0tomy need such indents?

Reply:
Nope, but I added the blank lines for clarity. I can remove them if u want.

Reply:
> I added the blank lines for clarity

I understand your intention, thank you. Let's go ahead with it as is.